openshift/template.yaml

```diff
@@ -0,0 +1,151 @@
+# Template to produce a new environment in OpenShift.
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: product-listings-manager-template
+
+parameters:
+- name: TAG
+  displayName: Application tag
+  description: Short unique identifier for object names
+  required: true
+- name: DOCKER_TAG
+  displayName: Docker tag
+  description: Docker tag from registry
+  required: true
+- name: FLASK_SECRET_KEY
```

Comment:
Do we need `FLASK_SECRET_KEY` here (yet :) ?

Reply:
I got the impression this is needed as in Django. Looks like it's required only when [sessions](http://flask.pocoo.org/docs/1.0/api/#sessions) are involved.

Removing for now.